x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/utils.ts

```diff
@@ -98,22 +112,47 @@ export const scrollAndBulkIndex = async (
     logger.warn('First bulk index was unsuccessful');
     return false;
   }
-  let newScrollId = someResult._scroll_id;
-  while (true) {
+
+  const totalHits = isTotalValueObject(someResult.hits.total)
+    ? someResult.hits.total.value
+    : someResult.hits.total;
```

Comment:
I think here you can use regular `typeof` and be ok?

I just tried it and it seems to work out. Would save some unit test writing around a new function:

```ts
  const totalHits =
    typeof someResult.hits.total === 'number' ? someResult.hits.total : someResult.hits.total.value;
```

Reply:
yup definitely better to just use typeof.